core/templates/dev/head/components/forms/HtmlSelectDirective.js

```diff
@@ -33,6 +33,15 @@ oppia.directive('htmlSelect', [function() {
       $scope.select = function(id) {
         $scope.selection = id;
       };
+
+      $scope.getSelectionIndex = function() {
+        for (var i = 0; i < $scope.options.length; i++) {
+          if (($scope.options[i].val === $scope.selection) ||
```

Comment:
So, one thought. For multiple choice, we pass {id: index, val: actual choice} if I'm not mistaken, and for image we pass {id: label, val: label}. Is this right? If so then we could always use id?